doxygen/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Finds and loads settings from an external project.
+# More information at https://cmake.org/cmake/help/v3.18/command/find_package.html
+find_package(Doxygen
+# The dot component is used for graph generation.
+# More information at https://cmake.org/cmake/help/v3.18/module/FindDoxygen.html
+	COMPONENTS dot)
+
```

Comment:
is this normal ?

Reply:
Mhh yeah I guess, it's a copy paste from doxygen doc I recall. Comments can be anywhere it seems ahah